MinGL.h

```diff
@@ -1,10 +1,25 @@
 #pragma once
+#include<math.h>
 
 struct GLFWwindow;
 
 struct MinGLColor
 {
 	float rgba[4] = { 0,0,0,0 };
+
+	float epsilon = 0.0039215;
+	bool operator == (const MinGLColor color2) const{
+		for(int i=0; i<4; i++){
+			if( fabs(this->rgba[i] - color2.rgba[i]) > epsilon )return false;
+		}
+		return true;
+	}
+	bool operator != (const MinGLColor color2) const{
+		for(int i=0; i<4; i++){
+			if( fabs(this->rgba[i] - color2.rgba[i]) > epsilon ) return true;
+		}
+		return false;
+	}
```

Comment:
Simplify to 
`return !(*this == color2);`

Reply:
done